values/velero/velero-raw.gotmpl

```diff
@@ -0,0 +1 @@
+resources: []
```

Comment:
If there is nothing to deploy here, we don't need this release at all. Please remove the `velero-artifacts` release.